src/software/ai/evaluation/time_to_travel_test.cpp

```diff
@@ -186,20 +162,20 @@ TEST_F(
     double final_vel   = 0.0;
     double max_accel   = 1.0;
     double max_vel     = 4.0;
-    double distance    = 1.0;
+    double dist_required_to_reach_v_f =
+        std::abs(std::pow(final_vel, 2) - std::pow(initial_vel, 2)) / (2 * max_accel);
+    double distance = dist_required_to_reach_v_f - 0.00001;
+    std::cout << "dist=" << distance << std::endl;
```

Comment:
debugging print out should be removed before merging

Reply:
good catch. removed